packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -29,6 +29,18 @@ enum FlexDirection {
   vertical
 }
 
+/// Which incoming constraint parameter defines how much free space is
+/// available along the main axis in a flex layout.
+enum MainAxisSpace {
+  /// The free space is the difference between incoming constraint's min parameter
+  /// and the collective size of the children, but not less than zero.
+  min,
+
+  /// The free space is the difference between incoming constraint's max parameter
+  /// and the collective size of the children, but not less than zero.
```

Comment:
+1 for `MainAxisSize`.  That name will probably also be useful for other layout models that have a notion of mainAxis and size but not of space.